modules/seqexec/server/src/main/scala/seqexec/server/ObserveActions.scala

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server
+
+import cats._
+import cats.effect.Sync
+import cats.effect.Concurrent
+import cats.data.Reader
+import cats.implicits._
+import edu.gemini.spModel.config2.Config
+import edu.gemini.spModel.obscomp.InstConstants._
+import edu.gemini.spModel.seqcomp.SeqConfigNames._
+import fs2.Stream
+import gem.Observation
+import org.log4s._
+import seqexec.engine._
+import seqexec.model.dhs.ImageFileId
+import seqexec.model.enum.ObserveCommandResult
+import seqexec.server.ConfigUtilOps._
+import seqexec.server.keywords._
+import seqexec.server.InstrumentSystem.ElapsedTime
+import seqexec.server.SeqTranslate._
+import squants.time.TimeConversions._
+
+/**
+ * Methods usedd to generate observation related actions
+ */
+trait ObserveActions {
```

Comment:
This trait contains methods previously on `SeqTranslate`. They will be used for a N&S aware `observe` function. Mostly they are copied verbatim from `SeqTranslate` with some improvements with respect to replace `IO` usages with `F[_]`